worker/worker.go

```diff
@@ -0,0 +1,60 @@
+package worker
+
+import (
+	"encoding/json"
+	"github.com/ufcg-lsd/arrebol-pb-worker/utils"
+	"io"
+	"log"
+)
+
+type Worker struct {
+	Vcpu           string
+	Ram            string
+	Token          string
+	Id             string
+	QueueId        string
```

Comment:
@paulofelipefeitosa @raonismaneoto do you guys think it is ok to have the QueueId in the worker struct? I have a bad feeling but if both of agree I'm ok with that

Reply:
The queueId stands for the queue whose worker has been attached by the server. In other words, it's the queue from which it will get tasks. It can also be set in deployment time, which means that the worker has been instantiated to get tasks from a specific queue only. I don't know where else we could keep this information, I think it's ok this way..

Reply:
I think that for now, it is ok, but if other queue attributes appear then it might time to rethink it.